internal/cli/atlas/quickstart/prompt.go

```diff
@@ -56,20 +56,21 @@ func newDBUsernameQuestion(dbUser, message string, validation func(val interface
 		Validate: validation,
 		Name:     "dbUsername",
 		Prompt: &survey.Input{
-			Message: message,
+			Message: fmt.Sprintf("Database User Username: %s", message),
 			Help:    usage.DBUsername,
 			Default: dbUser,
 		},
 	}
 	return q
 }
 
-func newDBUserPasswordQuestion() *survey.Question {
+func newDBUserPasswordQuestion(password, message string) *survey.Question {
 	return &survey.Question{
 		Name: "DBUserPassword",
-		Prompt: &survey.Password{
-			Message: "Insert the Password for authenticating to MongoDB [Press Enter to use an auto-generated password]",
+		Prompt: &survey.Input{
+			Message: fmt.Sprintf("Database User Password: %s", message),
```

Comment:
[not-blocking] this would put the optional text after the colon(:) I think it would be better to leave the colon(:) at the end

Reply:
same for the others except cluster